api/cluster-resources/aou-upload-policy-extension.js

```diff
@@ -0,0 +1,22 @@
+// Workbench "Upload policy" extension, to show the policy before letting user upload any file
+
+define([
+  'base/js/namespace'
+], (Jupyter) => {
+
+  const load_upload_extension = () => {
+  // This will open a dialog box when the user clicks Upload.
```

Comment:
Please specify which locations in the application this is guarding. I assume it's just for the file browser upload button.